### ISSUE TITLE
Issue about scannet gt mesh unification

### ISSUE THREAD

**Issue by PROPHETE-pro** [xhd0612/GaussianRoom#17]
Dear sir,
Thank you very much for your excellent work!
Here I have some questions want to ask for advice.
I notice that when dealing with scannet data, you seem to apply some adjustments to the ground truth mesh, making the self._xyz of the vertices stay within abs(x,y,z) < 0.5. I also write my own dataloader for scannet, but I do not apply this, so my self._xyz has some values larger than 5. This will resulting in doing D&P all the points will be purned, as 'sdf_densify_guidance =  self.gaussian_fun(sdf_val(self._xyz), self.get_opacity.squeeze())' will have extremely small values (e-32). Can you provide more details about how you process the scannet data? Thank you very much if you could help!

Besides, I also found that when processing my own scannet data( I already follow NeuRIS to process the data but no adjustment to the mesh), the grad of 'runner.renderer.sdf_network_fine.parameters()' will randomly become NaN after sdf_loss.backward(). I am not sure if this is also caused by the differece between your provided data and my own dataset.

**Comment by xhd0612** [comment]
Greetings!

Methods like NeuS and NeuRIS perform scene scale normalization. We have applied normalization to the ground truth mesh using the generated` trans_n2w.txt` file to facilitate geometric metric evaluation.

The results from NeuRIS are all in normalized coordinates, while 3DGS (Colmap) uses world coordinates. To convert between these coordinate systems, you need to use the transformation matrix stored in `trans_n2w.txt`.

Regarding the second issue you mentioned, I haven't encountered it myself and unfortunately don't have any insights. Perhaps you could reach out to the NeuRIS authors for help.
